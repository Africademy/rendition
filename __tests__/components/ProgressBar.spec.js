/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import {
  ProgressBar,
  Provider
} from '../../dist'

test('ProgressBar renders correctly', () => {
  const value = 50
  const component = renderer.create(
    <Provider>
      <ProgressBar value={value}>
        {value}%
      </ProgressBar>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
