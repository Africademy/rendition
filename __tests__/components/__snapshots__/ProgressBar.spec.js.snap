// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders correctly 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 bBubtL"
>
  <div
    className="sc-bdVaJa lgVSbN"
  >
    <div
      className="sc-gZMcBi eHXulF sc-iwsKbI jmZukC"
      color="#fff"
    >
      <div
        className="sc-htoDjs cNVTfd"
      >
        50
        %
      </div>
      <div
        className="sc-gzVnrw kYhhHL"
        style={
          Object {
            "width": "50%",
          }
        }
      >
        <div
          className="sc-dnqmqq flgyce"
          style={
            Object {
              "width": "200%",
            }
          }
        >
          50
          %
        </div>
      </div>
    </div>
  </div>
</div>
`;
